client/data/settings/hooks.js

```diff
@@ -75,3 +75,28 @@ export const useSettings = () => {
 		[ saveSettings ]
 	);
 };
+
+export const useDigitalWalletsSettings = () => {
+	const { updateIsDigitalWalletsEnabled } = useDispatch( STORE_NAME );
+	return useSelect( ( select ) => {
```

Comment:
very ⛏️ :D - just to be consistent with the `useDigitalWalletsLocations` added below and with the other hooks in the file
```suggestion
	const { updateIsDigitalWalletsEnabled } = useDispatch( STORE_NAME );

	return useSelect( ( select ) => {
```